coalib/bearlib/languages/Language.py

```diff
@@ -250,6 +281,12 @@ def __init__(self, *versions):
         else:
             self.versions = tuple(sorted(versions))
 
+    def __getattr__(self, item):
+        if len(self.versions) > 1:
```

Comment:
that's fine, you can only specify 0 if there are None and then you can't spec one. Which is good